src/utilities/time.ts

```diff
@@ -422,6 +422,48 @@ export function convertDurationStringToInterval(durationString: string): string
   return intervalString ? intervalString : '0';
 }
 
+/**
+ * Convert a Postgres UTC string to milliseconds, for rendering purposes.
+ * @param date The date as a string to convert.
+ * @returns The number of milliseconds since epoch start for this date.
+ */
+export function convertUTCtoMs(date: string): number {
+  const d = new Date(date);
+  return d.getTime();
+}
+
+/**
+ * Convert durations of format:
+ *  - "89:44:09.000000"
+ *  - "89:44:09":
+ *     + with no decimal point,
+ *     + or n <= 6 zeroes if point included.
+ * to milliseconds.
+ *
+ * @param duration the duration as a string
+ * @returns the number of milliseconds spanned by that duration.
+ */
+//
+export function convertDurationToMs(duration: string): number {
```

Comment:
Is this different from `getIntervalInMs`?

Reply:
Doesn't appear to be, replaced with `getIntervalInMs` and removed!

Reply:
Was this supposed to have been replaced with `getIntervalInMs`? It sounded like that's what was going to happen based on this convo, but it appears to still be being used.

Reply:
Ah, think the original commit got stepped on during a rebase - fixed now!